kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/WatchConnectionManager.java

```diff
@@ -293,7 +293,7 @@ public void onPong(Buffer buffer) {
 
       @Override
       public void onClose(final int code, final String reason) {
-        logger.info("WebSocket close received. code: {}, reason: {}", code, reason);
```

Comment:
WatchHTTPManager already has the normal path for open and close logging at debug level instead of info.  I'm looking at lines 111 and 272 in WatchHTTPManager.java